packages/concerto-types/src/generated/unions/concerto@1.0.0.ts

```diff
@@ -51,7 +59,11 @@ ILongDomainValidator |
 IAliasedType | 
 IImport | 
 IModel | 
-IModels;
+IModels | 
```

Comment:
Is this autogenerated or you've manually added?

Reply:
It is auto generated